Project.toml

```
@@ -3,7 +3,7 @@ uuid = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 keywords = ["markov chain monte carlo", "probablistic programming"]
 license = "MIT"
 desc = "A lightweight interface for common MCMC methods."
-version = "3.0.1"
+version = "4.0.0"
```

```suggestion
version = "3.0.2"
```

I think this should be considered part of the 3.0 break, since it's all the same general change.

---

Sounds good to me